quantized_mesh_encoder/ecef.py

```diff
@@ -16,22 +16,22 @@ def to_ecef(positions):
 
     Args:
         - positions: expected to be an ndarray with shape (-1, 3)
-    from latitude-longitude-height to
+        - ellisoid: a dict that defines an ellipsoid with with "a" and "b" values. Defaults to WGS84
+    from latitude-longitude-height to ecef
     """
-
     lon = positions[:, 0]
```

Comment:
I think above this an 
```py
assert isinstance(ellipsoid, Ellipsoid), 'ellipsoid must be an instance of Ellipsoid'
```
makes sense. After all, a `dict` won't work because `.a` and `.e2` won't work on a `dict`.

Reply:
I see you have it in `encode` below but I don't think it hurts to add to `to_ecef` as well. It looks like the `isinstance` call takes only about `50ns`, and an advanced user could use `to_ecef` outside of the `encode` call.